### DISCUSSION TITLE
已付费。建议重做注释颜色

### DISCUSSION THREAD

**Discussion by taolar** [MuiseDestiny/zotero-style#1160] [General]
付费用户，很支持p神的工作，先表示一下感谢
直奔主题，关于注释的颜色这方面，建议添加几个功能
1. 把删除，新增可以做到一个齿轮里面去
2. 可以把当前论文选择的那一套注释，高亮一下。
3. 在某一套注释里，每一行对应着每个颜色。希望能增加将每个颜色自定义顺序的操作。我如果想把灰色往前移，只能把前面的颜色删掉，等灰色自动掉到前面去了，再把删掉的颜色添加进来。但是这样的话也会使文献矩阵用不了。

<img width="798" alt="截屏2024-10-18 00 10 22" src="https://github.com/user-attachments/assets/7d3dbc8e-6eac-42c3-808a-21d536b0a9d8">

**Comment by MuiseDestiny** [comment]
https://www.bilibili.com/video/BV1AByiYfENg/?spm_id_from=333.999.0.0
